extensions/amp-story-shopping/0.1/amp-story-shopping-tag.js

```diff
@@ -1,30 +1,49 @@
 import {Layout, applyFillContent} from '#core/dom/layout';
 
+import {Services} from '#service';
+
+import {StateProperty} from '../../amp-story/1.0/amp-story-store-service';
+
 const TAG = 'amp-story-shopping-tag';
 
 export class AmpStoryShoppingTag extends AMP.BaseElement {
   /** @param {!AmpElement} element */
   constructor(element) {
     super(element);
 
-    /** @private {string} */
-    this.myText_ = TAG;
-
-    /** @private {?Element} */
-    this.container_ = null;
+    /** @private @const {!./amp-story-store-service.AmpStoryStoreService} */
+    this.storeService_ = null;
   }
 
   /** @override */
   buildCallback() {
     super.buildCallback();
-    this.container_ = this.element.ownerDocument.createElement('div');
-    this.container_.textContent = this.myText_;
-    this.element.appendChild(this.container_);
-    applyFillContent(this.container_, /* replacedContent */ true);
+    this.element.textContent = this.myText_;
+    applyFillContent(this.element, /* replacedContent */ true);
```

Comment:
We don't need `applyFillContent`. This element does not need the [styling applied from it](https://github.com/ampproject/amphtml/blob/main/build-system/tasks/create-golden-css/css/main.css#L209-L221).